WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailLikesView.swift

```diff
@@ -98,11 +98,11 @@ private extension ReaderDetailLikesView {
         case (true, 0):
             summaryLabel.attributedText = highlightedText(SummaryLabelFormats.onlySelf)
         case (true, 1):
-            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singularWithSelf, totalLikes))
+            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singularWithSelf))
```

Comment:
Since this localized string no longer has parameters or format specifiers in it, we could just use it like so:

```suggestion
            summaryLabel.attributedText = highlightedText(SummaryLabelFormats.singularWithSelf)
```